test_cachediff.py

```diff
@@ -35,7 +35,8 @@ def test_has_virtual_address(self):
 
 class TestFile(unittest.TestCase):
     def setUp(self):
-        file_path = os.path.join(os.getcwd(), 'test_file.c')
+        #file_path = os.path.join(os.getcwd(), 'test_file.c') since os.getcwd get /home/sameer etc
```

Comment:
why is it a problem if os.getcwd() returns /home/sameer? you want the full path to the file dont you ?